commerce_coordinator/apps/commercetools/tests/test_clients.py

```diff
@@ -803,6 +803,89 @@ def test_update_customer_with_anonymized_fields_exception(self):
 
                 log_mock.assert_called_once_with(expected_message)
 
+    def test_is_first_time_discount_eligible_success(self):
+        base_url = self.client_set.get_base_url_from_client()
+        email = 'email@example.com'
+        code = 'discount-code'
+
+        mock_orders = {
+            "total": 1,
+            "results": [
+                {
+                    "discountCodes": [
+                        {
+                            "discountCode": {
+                                "obj": {
+                                    "code": 'another-code'
+                                }
+                            }
+                        }
+                    ]
+                }
+            ]
+        }
```

Comment:
we can make it a constant since its been repeated across tests

Reply:
Both mock_orders have different code values hence, I didn't make it a constant